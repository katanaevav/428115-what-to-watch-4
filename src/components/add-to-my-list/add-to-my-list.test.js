import React from "react";
import renderer from "react-test-renderer";
import AddToMyList from "./add-to-my-list.jsx";

it(`Render add to My list button with isFavorite status`, () => {
  const tree = renderer.create(
      <AddToMyList
        isFavorite = {true}
        onButtonClick = {() => {}}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

it(`Render add to My list button without isFavorite status`, () => {
  const tree = renderer.create(
      <AddToMyList
        isFavorite = {false}
        onButtonClick = {() => {}}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
