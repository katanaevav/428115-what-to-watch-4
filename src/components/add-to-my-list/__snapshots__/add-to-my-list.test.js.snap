// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render add to My list button with isFavorite status 1`] = `
<button
  className="btn btn--list movie-card__button"
  onClick={[Function]}
  type="button"
>
  <svg
    height="14"
    viewBox="0 0 18 14"
    width="18"
  >
    <use
      xlinkHref="/sprite.svg#in-list"
    />
  </svg>
  <span>
    My list
  </span>
</button>
`;

exports[`Render add to My list button without isFavorite status 1`] = `
<button
  className="btn btn--list movie-card__button"
  onClick={[Function]}
  type="button"
>
  <svg
    height="20"
    viewBox="0 0 19 20"
    width="19"
  >
    <use
      xlinkHref="/sprite.svg#add"
    />
  </svg>
  <span>
    My list
  </span>
</button>
`;
